services/provider/server/server.go

```diff
@@ -390,33 +390,28 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 	})
 
 	if consumerResource.Spec.StorageQuotaInGiB > 0 {
-		clusterResourceQuotaSpec := &quotav1.ClusterResourceQuotaSpec{
-			Selector: quotav1.ClusterResourceQuotaSelector{
-				LabelSelector: &metav1.LabelSelector{
-					MatchExpressions: []metav1.LabelSelectorRequirement{
-						{
-							Key:      string(consumerResource.UID),
-							Operator: metav1.LabelSelectorOpDoesNotExist,
-						},
-					},
-				},
-			},
-			Quota: corev1.ResourceQuotaSpec{
-				Hard: corev1.ResourceList{"requests.storage": *resource.NewScaledQuantity(
-					int64(consumerResource.Spec.StorageQuotaInGiB),
-					resource.Giga,
-				)},
-			},
-		}
 
 		extR = append(extR, &pb.ExternalResource{
 			Name: "QuotaForConsumer",
 			Kind: "ClusterResourceQuota",
-			Data: mustMarshal(clusterResourceQuotaSpec),
+			Data: mustMarshal(getClusterResourceQuotaSpec(consumerResource)),
 		})
 
 	}
 
+	channelName, err := s.getOCSSubscriptionChannel(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get subscription channel %v", err)
+	}
+
+	extR = append(extR, &pb.ExternalResource{
+		Name: "operator-subs-channels",
+		Kind: "Subscription",
+		Data: mustMarshal(map[string]string{
+			"ocs-operator": channelName,
+		}),
```

Comment:
If the kind on the response is Subscription, I would expect to have a SubscriptionSpec sent back to the client.

But the main question is why this change is part of this PR in the first place. Let's leave any changes that are not tied directly to the hash mechanism for a follow-up PR